src/Codeception/Step.php

```diff
@@ -79,10 +79,17 @@ public function getAction()
         return $this->action;
     }
 
-    public function getLine()
+    public function getFilePath()
```

Comment:
`phpunit-wrapper` uses `getLine` in 2 places:
https://github.com/Codeception/phpunit-wrapper/search?q=getLine .

It would be the best to restore `getLine` method with original behaviour here and delete it in Codeception 5.0

Reply:
OK. I added a note there - but I probably won't remove it myself then...